src/redux/store.js

```diff
@@ -1,9 +1,17 @@
 import { createStore, combineReducers, compose, applyMiddleware } from 'redux';
-import thunk from 'redux-thunk';
```

Comment:
do you use `'redux-thunk'` anywhere else?
probably you need to remove it from the `package.json` also?

Reply:
Removed from package JSON